Detections/AWSCloudTrail/AWS_IngressEgressSecurityGroupChange.yaml

```diff
@@ -1,7 +1,8 @@
 ﻿id: 4f19d4e3-ec5f-4abc-9e61-819eb131758c
 name: Changes to AWS Security Group ingress and egress settings
 description: |
-  'Monitoring changes to AWS Security Group ingress and egress settings.
+  'A security group acts as a virtual firewall of an instance to control inbound and outbound traffic. 
```

Comment:
use Security Group instead of security group

Reply:
Done. Thanks